File: android/app/src/main/java/voidawt/image/ImageObserver.java
package voidawt.image;

public interface ImageObserver {
    int WIDTH = 1;
    int HEIGHT = 2;
    int PROPERTIES = 4;
    int SOMEBITS = 8;
    int FRAMEBITS = 16;
    int ALLBITS = 32;
    int ERROR = 64;
    int ABORT = 128;

    boolean imageUpdate(voidawt.Image img, int infoflags, int x, int y, int width, int height);
}

File: src/commonMain/kotlin/awt/image/ImageObserver.kt
package awt.image

import awt.Image

expect interface ImageObserver {
    fun imageUpdate(img: Image, infoflags: Int, x: Int, y: Int, width: Int, height: Int): Boolean
}

expect val WIDTH: Int
expect val HEIGHT: Int
expect val PROPERTIES: Int
expect val SOMEBITS: Int
expect val FRAMEBITS: Int
expect val ALLBITS: Int
expect val ERROR: Int
expect val ABORT: Int